waimak-core/src/main/scala/com/coxautodata/waimak/dataflow/DataFlow.scala

```diff
@@ -336,6 +341,20 @@ trait DataFlow[Self <: DataFlow[Self]] extends Logging {
     allLabels.diff(allLabels.distinct).toSet
   }
 
+  private[dataflow] def getEnabledConfigurationExtensions: Seq[DataFlowConfigurationExtension[Self]] = {
+    import scala.reflect.runtime.currentMirror
+    import scala.reflect.runtime.universe.typeOf
+
+    val needType = typeOf[DataFlowConfigurationExtension[Self]]
+    ServiceLoader
+      .load(classOf[DataFlowConfigurationExtension[Self]])
+      .asScala
+      .filter(currentMirror.reflect(_).symbol.toType <:< needType)
+      .filter(e => flowContext.getBoolean(s"${EXTENSIONS_PREFIX}.${e.extensionKey}.enabled", false))
+      .toSeq
+
```

Comment:
No exception is thrown if an extension was enabled but it wasn't found as a service. This means that extensions would be silently ignored if they could not be found. What do you think should be the right behaviour @vavison ?

Reply:
This is done now